index.html

```diff
@@ -195,12 +195,44 @@ <h2 class="card-text2">Skills</h2>
 
 
    <!-- get in touch -->
-   <div class="contact">
-     <p class="contact-info">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
 
-   </div>
+   <div class="contact-container">
+     <div class="contact-message">
+       <p class="contact-info">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
```

Comment:
- [ ] Kindly add a little margin to this text according to the design template. Thanks 😄